pkg/input/input.go

```diff
@@ -18,9 +17,9 @@ type InputConfig struct {
 
 // SeriesParams determines what data should be loaded from the input.
 type SeriesParams struct {
-	Matcher string
-	MinTime thanosmodel.TimeOrDurationValue
-	MaxTime thanosmodel.TimeOrDurationValue
+	Matchers []*labels.Matcher
```

Comment:
@bwplotka @iNecas Do we know of a way to translate this `labels.Matcher` ([here](https://pkg.go.dev/github.com/prometheus/prometheus/pkg/labels@v1.8.2-0.20200629082805-315564210816?tab=doc#Matcher)) type to `prompb.LabelMatcher` ([here](https://pkg.go.dev/github.com/prometheus/prometheus/prompb@v1.8.2-0.20200629082805-315564210816?tab=doc#LabelMatcher))? or should I try to individually convert it?

Reply:
nvm I found the function I need in storepb (https://github.com/thanos-io/thanos/blob/v0.15.0/pkg/store/storepb/custom.go#L450)

Reply:
Yes (: